.eslintignore

```diff
@@ -4,4 +4,5 @@ webpack.*.js
 # tests
 src/Frontend/test/**/*.js
 src/Frontend/config/*.js
+src/Frontend/config/*/*.js
```

Comment:
can be simplified to ``src/Frontend/config/*``, but we should figure out whether that's necessary